client/main.lua

```diff
@@ -386,6 +386,18 @@ local function preventVehicleFlip()
     end
 end
 
+local function preventAirControl()
+    local veh = GetVehiclePedIsIn(cache.vehicle, false)
+    if not (DoesEntityExist(veh) and not IsEntityDead(veh)) then
+        local model = GetEntityModel(veh)
+        -- If it's not a boat, plane or helicopter, and the vehicle is off the ground with ALL wheels, then block steering/leaning left/right/up/down.
+        if IsThisModelABoat(model) or IsThisModelAHeli(model) or IsThisModelAPlane(model) or not IsEntityInAir(veh) then
+            DisableControlAction(0, 59) -- leaning left/right
+            DisableControlAction(0, 60) -- leaning up/down
+        end
+    end
+end
```

Comment:
```suggestion
local function preventAirControl()
    if not cache.vehicle or IsEntityDead(cache.vehicle) then return end

    local model = GetEntityModel(cache.vehicle)
    -- If it's not a boat, plane or helicopter, and the vehicle is off the ground with ALL wheels, then block steering/leaning left/right/up/down.
    if IsThisModelABoat(model) or IsThisModelAHeli(model) or IsThisModelAPlane(model) or not IsEntityInAir(cache.vehicle) then return end

    DisableControlAction(0, 59, true) -- leaning left/right
    DisableControlAction(0, 60, true) -- leaning up/down
end
```
I'm adding the IsEntityDead as well, but I'm not sure if that works on vehicles